src/ReplicatedStorage/Client/InterfaceComponents/Toolbar/init.lua

```diff
@@ -0,0 +1,254 @@
+local UserInputService = game:GetService("UserInputService")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Packages = ReplicatedStorage:WaitForChild("Packages")
+
+local Trove = require(Packages.Trove)
+local Fusion = require(Packages.Fusion)
+local WrappedFusion = require(Packages.FusionUtils).Wrapped
+
+local UIComponents = script.Parent
+local ToolbarItem = require(UIComponents.Toolbar.ToolbarItem)
+
+local New = WrappedFusion.New
+local Statify = WrappedFusion.Statify
+local Value = Fusion.Value
+local Children = Fusion.Children
+local Computed = Fusion.Computed
+local Spring = Fusion.Spring
+local ForValues = Fusion.ForValues
+local Cleanup = Fusion.Cleanup
+local Observer = Fusion.Observer
+
+local KEY_CODES = {
+	Enum.KeyCode.One,
+	Enum.KeyCode.Two,
+	Enum.KeyCode.Three,
+	Enum.KeyCode.Four,
+	Enum.KeyCode.Five,
+	Enum.KeyCode.Six,
+	Enum.KeyCode.Seven,
+	Enum.KeyCode.Eight,
+	Enum.KeyCode.Nine,
+	Enum.KeyCode.Zero,
+}
+
+local function keyCodeToString(keyCode)
+	if keyCode == "" then
+		return ""
+	elseif keyCode.Value < 127 and keyCode.Value > 33 then
+		return string.char(keyCode.Value)
+	else
+		return keyCode.Name
+	end
+end
```

Comment:
Can I get some explanation on why keycodes need to be converted to strings first?

Reply:
Additionally, why are values 34-126 special?

Reply:
Explained this in discord, resolving this conversation.

Basically: KeyCode.Value within range returns ascii characters. Roblox stores keycode names as ".One", we need "1"